packages/strapi-plugin-content-manager/admin/src/containers/EditView/Header.js

```diff
@@ -28,23 +29,28 @@ const Header = () => {
     resetData,
     setIsSubmitting,
     slug,
+    clearData,
   } = useDataManager();
+  const isSingleType = pathname.split('/')[3] === 'singleType';
```

Comment:
Here since the base url is is `/plugins/content-manager/:type` you can retrieve the type by using [this hook](https://reacttraining.com/react-router/web/api/Hooks/useroutematch) instead of parsing the url.